### PR TITLE
set expectation on how long install and commit will take

### PR DESCRIPTION
**Pull request by depombo**
Untitled 2

### CODE REVIEW

File: dashboard/src/components/QueryResult.tsx [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+  const query = editorTabs[editorSelectedTab]?.content;
+  const longRunningRpc = query.includes('iasql_commit()') || query.includes('iasql_install(');
 
   // TODO: show all statements with its respective query and not just the last one
   return editorTabs?.[editorSelectedTab]?.isRunning ? (
-    <Spinner />
+    longRunningRpc ? (
+      <VBox>
+        <p className='pt-20 px-20 font-bold'>
+          We are importing the entire state of the infrastructure from the cloud account and due to default
```
Comment (aguillenv) [inline dashboard/src/components/QueryResult.tsx:16]: maybe the `We are importing the entire state...` does not align 100% with `iasql_commit`?

Comment (depombo) [reply dashboard/src/components/QueryResult.tsx:16]: technically we do a sync on commit, but yes. perhaps `reading` or `parsing`?

Comment (aguillenv) [reply dashboard/src/components/QueryResult.tsx:16]: What about: `Cloud operations can take several minutes due to default cloud SDK rate limiters depending on how many resources are in the account.  Please be patient.`

**Review by aguillenv — APPROVED** [review]
looks good to me! Maybe update the text but not blocking

**Review by dfellis — APPROVED** [review]

**Pull request merged**